libs/filabridge/include/private/filament/SibStructs.h

```diff
@@ -42,6 +42,12 @@ struct PerRenderPrimitiveMorphingSib {
     static constexpr size_t SAMPLER_COUNT  = 2;
 };
 
+struct PerRenderPrimitiveSkinningSib {
+    static constexpr size_t BONE_IaW       = 0;   //bone indices and weights
```

Comment:
rename to `BONE_INDICES_AND_WEIGHTS`

Reply:
I think we coud simplify things here by moving this into the `PerRenderPrimitiveMorphingSib` because they both use the same variant anyways (and that's not going to change).
(The name would be wrong, but we could fix that in a later PR).
I think that would simplify a lot of the changes to libfilamat.

Reply:
It's good idea. I thought about it but wasn't sure if I should make this change